test/Integration/SqlInputBindingIntegrationTests.cs

```diff
@@ -35,10 +35,11 @@ public async void GetProductsTest(int n, int cost, SupportedLanguages lang)
             HttpResponseMessage response = await this.SendInputRequest("getproducts", cost.ToString());
 
             // Verify result
-            string expectedResponse = JsonConvert.SerializeObject(products);
+            Product[] expectedResponse = products;
```

Comment:
Don't really need this extra var - can just use products below directly (same for similar situations below)